README.md

```diff
@@ -1,24 +1,18 @@
 # Lucky's Website
 
-### Future guide goals
-
-- Add screencasts
-- _Versioned_. Look at old ones, or even look at what's on master
-- Link to relevant screencasts
-- Helpful or not button
-  - If not, direct to Discord to get help
-- Check code snippets in real projects. More confidence code samples work.
```

Comment:
we have screencasts linked through LuckyCasts, and some of these are already created as issues.